helm_deploy/hmpps-breach-notice-api/values.yaml

```
@@ -35,7 +35,8 @@ generic-service:
       HMPPS-AUTH_CLIENT-ID: CLIENT_CREDS_CLIENT_ID
       HMPPS-AUTH-SECRET: CLIENT_CREDS_CLIENT_SECRET
     hmpps-domain-events:
-      HMPPS_SQS_TOPICS_HMPPSBREACHNOTICEPUBLISHTOPIC: topic_arn
+      HMPPS_SQS_TOPICS_HMPPSBREACHNOTICEPUBLISHTOPIC_ARN: topic_arn
+      HMPPS_SQS_TOPICS_HMPPSBREACHNOTICETOPIC_ARN: topic_arn
```

The API will read from a queue (which has a subscription to the topic) rather than the topic itself, so this should be something like:
```suggestion
    sqs-queue:
      HMPPS_SQS_QUEUES_HMPPSBREACHNOTICEQUEUE_QUEUE_NAME: sqs_queue_name
    sqs-dlq:
      HMPPS_SQS_QUEUES_HMPPSBREACHNOTICEQUEUE_DLQ_NAME: sqs_queue_name
```